docs/assetmanager.md

```diff
@@ -0,0 +1,26 @@
+# AssetManager
+
+AssetManager manages asset bundle configuration and loading.
+
+## 1.- Public Methods:
```

Comment:
Should be moved to phpdoc. No need to define it here.

Reply:
Remove. done